File: app/src/main/java/com/soho/sohoapp/network/results/PropertyFinanceResult.java
package com.soho.sohoapp.network.results;

import com.google.gson.annotations.SerializedName;

public class PropertyFinanceResult {
    @SerializedName("id")
    public int id;

    @SerializedName("purchase_price")
    public int purchasePrice;

    @SerializedName("loan_amount")
    public int loanAmount;

    @SerializedName("estimated_value")
    public int estimatedValue;

    @SerializedName("is_rented")
    public boolean isRented;

    @SerializedName("estimated_rent")
    public int estimatedRent;

    @SerializedName("actual_rent")
    public int actualRent;

    @SerializedName("leased_to")
    public String leasedTo;
}

File: app/src/main/java/com/soho/sohoapp/data/dtos/PropertyFinanceResult.kt
package com.soho.sohoapp.data.dtos

import com.google.gson.annotations.SerializedName

class PropertyFinanceResult {
    @SerializedName("id")
    var id: Int = 0

    @SerializedName("purchase_price")
    var purchasePrice: Double = 0.0

    @SerializedName("loan_amount")
    var loanAmount: Double = 0.0

    @SerializedName("estimated_value")
    var estimatedValue: Double = 0.0

    @SerializedName("is_rented")
    var isRented: Boolean = false

    @SerializedName("estimated_rent")
    var estimatedRent: Double = 0.0

    @SerializedName("actual_rent")
    var actualRent: Double = 0.0

    @SerializedName("leased_to")
    var leasedTo: String? = null
}
